Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM scratch
+FROM ubuntu:14.04
```

Comment:
The problem is that these minimal images are missing several libs like libpthread, libdevmapper etc. which are not missing by defaut when a ubuntu baseimage is being used.